README.md

```diff
@@ -6,8 +6,7 @@ install Python 3.4
 install pip
 
 pip install virtualenv
-virtualenv -p /usr/bin/python3.4 virtualenv
-(måske skal stien til python rettes)
+virtualenv -p $(which python) virtualenv
```

Comment:
The required version is already in the [README.md at line 5: install Python 3.4
](https://github.com/CodingPirates/forenings_medlemmer/blob/7b8b77523014e4c3558c6bdc8a5a252f6af69fb4/README.md#L5)